dali/dfu/dfuutil.cpp

```diff
@@ -857,21 +857,83 @@ class CDFUhelper: public CInterface, implements IDFUhelper
             else
                 throwError1(DFUERR_DFileNotFound, lfn);
         }
-        else {
-            Owned<IFileDescriptor> fdesc = file->getFileDescriptor();
-            Owned<IPropertyTree> t = fdesc->getFileTree();
+        else
+        {
+            Owned<IPropertyTree> t = queryDistributedFileDirectory().getFileTree(lfn, user, NULL, FOREIGN_DALI_TIMEOUT, true, true);
```

Comment:
>FOREIGN_DALI_TIMEOUT

clearer to go with the defaults here, as no foreign dali involved.
so clearer if change to:
```
Owned<IPropertyTree> t = queryDistributedFileDirectory().getFileTree(lfn, user);
```